confidential-data-hub/hub/src/bin/ttrpc-cdh-tool.rs

```diff
@@ -105,6 +108,15 @@ struct PullImageArgs {
     bundle_path: String,
 }
 
+#[derive(Debug, Args)]
+#[command(author, version, about, long_about = None)]
+struct InitOverlayNetworkArgs {
+    #[arg(short, long)]
+    pod_name: String,
+    #[arg(short, long)]
+    lighthouse_pub_ip: String,
```

Comment:
I think this could maybe come from the CDH config file (with init-data coming soon) rather than be passed from the caller.